docs/workflow/trimming/feature-switches.md

```diff
@@ -16,6 +16,7 @@ configurations but their defaults might vary as any SDK can set the defaults dif
 | UseSystemResourceKeys | System.Resources.UseSystemResourceKeys |  Any localizable resources for system assemblies is trimmed when set to true |
 | HttpActivityPropagationSupport | System.Net.Http.EnableActivityPropagation | Any dependency related to diagnostics support for System.Net.Http is trimmed when set to false |
 | StartupHookSupport | System.StartupHookProvider.IsSupported | Startup hooks are disabled when set to false. Startup hook related functionality can be trimmed. |
+| AllowBinaryFormatterUsageInTypeConverterSerialization | System.ComponentModel.TypeConverter.AllowUnsafeBinaryFormatterSerialization | BinaryFormatter serialization support is trimmed when set to false. |
```

Comment:
We don't use the word `Allow` in any other switch. Maybe we should use `EnableUnsafe` as a prefix? Also I think `TypeConverter` is too broad.

`AllowBinaryFormatterUsageInTypeConverterSerialization` => `EnableUnsafeBinaryFormatterDesigntimeLicenseContextSerialization` ?

Reply:
Changed it to `EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization`